src/NServiceBus.Core/IMessageSession.cs

```diff
@@ -43,14 +43,14 @@ public interface IMessageSession
         /// This method is only necessary if you turned off auto-subscribe.
         /// </summary>
         /// <param name="eventType">The type of event to subscribe to.</param>
-        /// <param name="options">Options for the subscribe.</param>
-        Task Subscribe(Type eventType, SubscribeOptions options);
```

Comment:
Change the PublishOptions in the Publish<T>  method above as well to be consistent?